Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -1776,39 +1774,59 @@ def getResultSummary(self):
         return {u'step': status}
 
 
-class DisplayUnexpectedResults(buildstep.BuildStep, AddToLogMixin):
-    name = 'display-unexpected-results'
+class DisplaySmartPointerResults(buildstep.BuildStep, AddToLogMixin):
+    name = 'display-smart-pointer-results'
 
     @defer.inlineCallbacks
     def run(self):
-        result_directory = f"public_html/results/{self.getProperty('buildername')}/{self.getProperty('archive_revision')} ({self.getProperty('buildnumber')})"
-        unexpected_results_json = os.path.join(result_directory, SCAN_BUILD_OUTPUT_DIR, 'unexpected_results.json')
+        self.zipFile = f"public_html/results/{self.getProperty('buildername')}/{self.getProperty('archive_revision')} ({self.getProperty('buildnumber')}).zip"
```

Comment:
This path looks similar to  self.resultDirectory. Do you want to just re-use the path from that variable to avoid duplication? something like:
`self.zipFile = f"{self.resultDirectory}.zip"`